### PR TITLE
feat: support tileset history

### PR DESCRIPTION
**Pull request by blacha**
Every tileset change now creates a history object, the current "production" tag is what is by default rendered to users.

using a url `/aerial@head/` will allow testers to gain access to early access imagery

### CODE REVIEW

File: packages/lambda-shared/src/aws/__test__/tile.metadata.table.test.ts [hunk 1]
```diff
@@ -1,157 +1,159 @@
...
-        o(ans.get('im_199')).deepEquals({
-            id: 'im_199',
-            projection: EPSG.Google,
-            year: 2001,
-            resolution: 100,
-            quadKeys: ['313'],
-        } as TileMetadataImageryRecord);
-    });
-});
+// import { EPSG } from '@basemaps/geo';
```
Comment (blacha) [inline packages/lambda-shared/src/aws/__test__/tile.metadata.table.test.ts:1]: TODO

File: packages/cog/src/cli/basemaps/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Basemaps Configuration utility
+
+This tool is designed to configure the basemaps rendering engine prority.
+
+### TileSets
+
+To render a basemap, a collection of imagery needs to be configured, basemaps uses a 'TileSet' for this configuration.
+
+A tileset is a list of imgagery along with the zoom levels to render at and the priority for rendering. The higher the number, the later the imagery will be layered onto the canvas. So bigger number means it will be ontop when viewed.
```
Comment (jacott) [inline packages/cog/src/cli/basemaps/README.md:9]: > imgagery

mispelling

File: packages/cog/src/cli/basemaps/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+The entire history of the tileset is stored inside the metadata database and version tags are used to configure what is rendered to the user
+
+Common tags:
+
+-   Head - The most recent tile set configuration
+-   Production - The tile set configuration which is rendered by default to the user
```
Comment (jacott) [inline packages/cog/src/cli/basemaps/README.md:16]: These appear to be lower case in the code. Suggest writing them here as `head` and `production`

File: packages/lambda-xyz/src/cli/import.imagery.ts [hunk 4]
```diff
@@ -67,16 +68,16 @@ async function main(): Promise<void> {
             maxZoom: mosaic.zoom.max,
             minZoom: mosaic.zoom.min,
         });
-        logger.info({ record: record.id }, 'Insert');
+        // logger.info({ record: record.id }, 'Insert');
 
-        await Aws.tileMetadata.db.create(record);
+        // await Aws.tileMetadata.put(record);
     }
```
Comment (jacott) [inline packages/lambda-xyz/src/cli/import.imagery.ts:74]: Why are these commented out?

Comment (blacha) [reply packages/lambda-xyz/src/cli/import.imagery.ts]: I have uncommented them

**Comment by palmerj** [comment]
Very Nice!

File: packages/lambda-shared/src/aws/__test__/tile.metadata.table.test.ts (hunk 1 above)
Comment (jacott) [reply packages/lambda-shared/src/aws/__test__/tile.metadata.table.test.ts]: Still TODO

File: packages/lambda-xyz/src/cli/import.imagery.ts [hunk 5]
```diff
@@ -67,16 +68,16 @@ async function main(): Promise<void> {
             maxZoom: mosaic.zoom.max,
             minZoom: mosaic.zoom.min,
         });
-        logger.info({ record: record.id }, 'Insert');
+        // logger.info({ record: record.id }, 'Insert');
 
-        await Aws.tileMetadata.db.create(record);
+        // await Aws.tileMetadata.put(record);
```
Comment (jacott) [inline packages/lambda-xyz/src/cli/import.imagery.ts]: Remove these lines?

**Pull request merged**